.github/workflows/publish-to-pypi.yaml

```diff
@@ -11,7 +11,7 @@ jobs:
 
     environment:
       name: pypi
-      url: https://test.pypi.org/p/pydantic2zod
+      url: https://pypi.org/p/pydantic2zod
```

Comment:
It did publish, so this may be redundant: https://pypi.org/project/pydantic2zod/